src/commands.rs

```diff
@@ -320,6 +320,14 @@ pub struct ElectrumOpts {
         default_value = "10"
     )]
     pub stop_gap: usize,
+
+    /// Enable domain validation when connecting to Electrum servers.
+    #[clap(
+        name = "VALIDATE_DOMAIN",
+        long = "validate_domain",
+        default_value = "true"
+    )]
+    pub validate_domain: bool,
```

Comment:
Change type to Option:
```suggestion
    pub validate_domain: Option<bool>,
```

Reply:
Done.